### PR TITLE
doc: known_issues: Add NCSDK-16060

### PR DESCRIPTION
**Pull request by MarekPieta**
Change adds NCSDK-16060 to known issues.

Jira: NCSDK-16060

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -1138,6 +1138,13 @@ Bluetooth® LE
 
 .. rst-class:: v2-0-0
 
+NCSDK-16060: :ref:`peripheral_lbs` sample build fails when :kconfig:option:`CONFIG_BT_LBS_SECURITY_ENABLED` is disabled
```
Comment (KAGA164) [inline doc/nrf/known_issues.rst:1141]:
```suggestion
NCSDK-16060: :ref:`peripheral_lbs` sample build fails when the :kconfig:option:`CONFIG_BT_LBS_SECURITY_ENABLED` is disabled
```

File: doc/nrf/known_issues.rst [hunk 2]
```diff
@@ -1138,6 +1138,13 @@ Bluetooth® LE
...
+  Build failure is caused by undefined ``conn_auth_info_callbacks`` structure.
```
Comment (KAGA164) [inline doc/nrf/known_issues.rst:1142]:
```suggestion
  Build failure is caused by undefined :c:struct:`conn_auth_info_callbacks` structure.
```

File: doc/nrf/known_issues.rst [hunk 3]
```diff
@@ -1138,6 +1138,13 @@ Bluetooth® LE
 
 .. rst-class:: v2-0-0
 
+NCSDK-16060: :ref:`peripheral_lbs` sample build fails when the :kconfig:option:`CONFIG_BT_LBS_SECURITY_ENABLED` is disabled
+  Build failure is caused by undefined ``conn_auth_info_callbacks`` structure.
```
Comment (divipillai) [inline doc/nrf/known_issues.rst:1142]:
```suggestion
NCSDK-16060: :ref:`peripheral_lbs` sample build fails when the :kconfig:option:`CONFIG_BT_LBS_SECURITY_ENABLED` option is disabled
  Build failure is caused by the undefined ``conn_auth_info_callbacks`` structure.
```

**Review by divipillai — APPROVED** [review]
Minor comments.

File: doc/nrf/known_issues.rst (hunk 2 above)
Comment (KAGA164) [reply doc/nrf/known_issues.rst]: Ok

**Review by KAGA164 — APPROVED** [review]

**Pull request merged**